explorer/trace_testdata/full_trace.carbon

```diff
@@ -93,18 +93,18 @@ fn Main() -> i32 {
 // CHECK:STDOUT: checking BindingPattern n: i32
 // CHECK:STDOUT: checking ExpressionPattern i32
 // CHECK:STDOUT: checking IntTypeLiteral i32
-// CHECK:STDOUT: (+) stack-push: i32 .0.
-// CHECK:STDOUT: (+) stack-push: i32 .0.
+// CHECK:STDOUT: (+) stack-push: pos-0 ValueExpressionAction `i32` (full_trace.carbon:11)
+// CHECK:STDOUT: (+) stack-push: pos-0 ExpressionAction `i32` (full_trace.carbon:11)
 // CHECK:STDOUT: --- step exp i32 .0. (full_trace.carbon:11) --->
```

Comment:
It seems like the code generating this line should be updated to use `Action::Print` for consistency. Note that you'll also need to update `explorer/README.md` to reflect the new trace format.

Reply:
I was thinking of updating the interpreter related trace output in a separate PR.
Yes, I'll update the documentation for the new format. Thanks for pointing it out!

Reply:
I'd prefer not to introduce an inconsistency, even temporarily. Would updating the `--- step` output in this PR create other inconsistencies?